vector/src/main/java/im/vector/app/features/crypto/keys/KeysExporter.kt

```diff
@@ -34,7 +34,7 @@ class KeysExporter @Inject constructor(
     suspend fun export(password: String, uri: Uri) {
         withContext(dispatchers.io) {
             val data = session.cryptoService().exportRoomKeys(password)
-            context.contentResolver.openOutputStream(uri)
+            context.contentResolver.openOutputStream(uri, "wt")
```

Comment:
Nit: maybe creating a quick extension like this could help to avoid code duplication:

```kotlin
    fun Context.safeOpenOutputStream(uri: Uri): OutputStream? {
        return contentResolver.openOutputStream(uri, "wt")
    }
```

Reply:
:+1: 
Done